src/entities/track/common/ui/TrackEdit/TrackEdit.tsx

```diff
@@ -43,22 +47,30 @@ export const TrackEdit = ({
     duration: validateHumanReadableDuration(fields.duration) ? undefined : message('form.invalid.format'),
   });
 
-  const submit = (fields: TTrackInputEditForm, formApi: FormApi<TTrackInputEditForm, typeof initialValues>) => {
+  const submit = async (fields: TTrackInputEditForm, formApi: FormApi<TTrackInputEditForm, typeof initialValues>) => {
     const durationISO = humanReadableDurationToISO(fields.duration);
 
     const formHadChanges = formApi.getState().dirty;
 
     if (durationISO && formHadChanges) {
-      updateTrack(
-        {
-          start: fields.start,
-          comment: fields.comment?.trim(),
-          duration: durationISO,
-        },
-        issueKey,
-        trackId,
-      );
+      try {
+        await updateTrack(
+          {
+            start: fields.start,
+            comment: fields.comment?.trim(),
+            duration: durationISO,
+          },
+          issueKey,
+          trackId,
+        );
+      } catch (error) {
+        if (error && typeof error === 'object' && 'message' in error) {
+          return { comment: error.message };
+        }
```

Comment:
```suggestion
        }
        throw error;
```
Точно не нужно выбросить ошибку, если это неизвестное? Тихо пропускаем?